app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
+
+<%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email) %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Resend confirmation instructions" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] [REQUIRED] Hello Team, great job with the devise configuration, however, it seems the configuration was not successfully done because after signup I am bee told to confirm my email before I can log in but that throws an error while I try to confirm.
<img width="742" alt="Screenshot 2023-04-28 at 21 59 23" src="https://user-images.githubusercontent.com/58692590/235255524-0fd87e3a-e96f-4677-9ce1-0a13a180d9b7.png">
To use this feature, kindly ensure that confirmable is been enabled.  Confirmable functionality requires the `:confirmable` module to be included.
Please enable `:confirmable` in the User model.

https://github.com/EddyShimwa/Recipe_app/blob/59b4fbd3d0c82b5290cf476bb9f9e505fc35b852/app/models/user.rb#L3